apps/masterbots.ai/app/api/chat/actions/actions.tsx

```
@@ -1,295 +1,307 @@
-    ] as ChatCompletionMessageParam[]
-    const processedMessages = setStreamerPayload(clientType, messages)
+      { role: "user", content: prompt },
+    ] as ChatCompletionMessageParam[];
+    const processedMessages = setStreamerPayload(clientType, messages);
 
     const response = await createResponseStream(clientType, {
       model: AIModels.Default,
-      messages: processedMessages
-    } as any)
+      messages: processedMessages,
+    } as any);
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)